streams/src/test/java/org/apache/kafka/streams/state/internals/metrics/RocksDBMetricsRecorderTest.java

```diff
@@ -557,10 +555,10 @@ public void shouldNotRecordStatisticsBasedMetricsIfStatisticsIsNull() {
 
     @Test
     public void shouldCorrectlyHandleHitRatioRecordingsWithZeroHitsAndMisses() {
-        resetToNice(statisticsToAdd1);
+        reset(statisticsToAdd1);
         recorder.addValueProviders(SEGMENT_STORE_NAME_1, dbToAdd1, cacheToAdd1, statisticsToAdd1);
         expect(statisticsToAdd1.getHistogramData(anyObject())).andStubReturn(new HistogramData(0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0L, 0L, 0.0));
-        expect(statisticsToAdd1.getTickerCount(anyObject())).andStubReturn(0L);
```

Comment:
`getTickerCount()` is never called in production code. You can comment out this line and the test will still be successful. This is because the mock is specified to be nice.